lib/api/v1/controllers/results.go

```diff
@@ -0,0 +1,15 @@
+package controllers
+
+import (
+	"github.com/gin-gonic/gin"
+)
+
+type ResultsController struct {
+	BaseController
+}
+
+func (c *ResultsController) Create(ctx *gin.Context) {
+	c.EnsureAuthenticatedUser(ctx)
+
+	// TODO: Work on file upload in anotehr PR
```

Comment:
```suggestion
	// TODO: Work on file upload in another PR
```

Reply:
Thanks, fixed in [d63d2f8](https://github.com/carryall/go-google-scraper-challenge/pull/93/commits/d63d2f818545444077139b243945bd6a29fe6361) 😅